content/about-us/hackathons.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "About NWB Hackathons"
```

Comment:
Maybe rename to "About NWB Events"? since this include roadshows, workshops, and other training events now.